src/api/app/queries/workflow_runs_finder.rb

```diff
@@ -13,7 +13,7 @@ def group_by_event_type
     end
   end
 
-  def with_event_type(event_type, action=nil)
+  def with_event_type(event_type, action = nil)
```

Comment:
```suggestion
  def with_event_type(event_type, request_action = nil)
```
To be consistent with the rest of the changes